Sources/Common/DataModel/SelectionNode/index.d.ts

```diff
@@ -54,7 +54,7 @@ export interface vtkSelectionNode extends vtkObject {
 	/**
 	 * This functions is called internally by VTK.js and is not intended for public use.
 	 */
-	setProperties(properties: ISelectionNodeProperties);
+	setProperties(properties: ISelectionNodeProperties):any;
```

Comment:
```suggestion
	setProperties(properties: ISelectionNodeProperties): boolean;
```

This is an auto-generated setter, so it returns boolean. (Returns true if the underlying value is actually changed).